template/test/unit/.eslintrc

```diff
@@ -1,6 +1,7 @@
 {
   "env": {
-    "mocha": true
+    "mocha": true,
+    "browser": true
```

Comment:
Yes, ESLint does not allow for globals to be used. As it is dependent on the environment wether the globals exist or not. Something ESLint cannot know about within the IDE or CLI. So we need to tell it manually browser globals can be used, which is save since karma always runs in a browser.